2021-09-26 11:26:16,960 ERROR: Exception on /index [GET] [in c:\users\anton\downloads\lib\site-packages\flask\app.py:1454]
Traceback (most recent call last):
  File "c:\users\anton\downloads\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\anton\downloads\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\anton\downloads\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\anton\downloads\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "c:\users\anton\downloads\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\routes.py", line 53, in index
    return render_template('index.html', title=_('Home'), form=form,
  File "c:\users\anton\downloads\lib\site-packages\flask\templating.py", line 147, in render_template
    return _render(
  File "c:\users\anton\downloads\lib\site-packages\flask\templating.py", line 128, in _render
    rv = template.render(context)
  File "c:\users\anton\downloads\lib\site-packages\jinja2\environment.py", line 1304, in render
    self.environment.handle_exception()
  File "c:\users\anton\downloads\lib\site-packages\jinja2\environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\templates\index.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\anton\downloads\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\anton\downloads\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block 'doc'
    {%- block html %}
  File "c:\users\anton\downloads\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block 'html'
    {% block body -%}
  File "c:\users\anton\downloads\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 23, in block 'body'
    {% block content -%}
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\templates\base.html", line 48, in block 'content'
    {% block app_content %}{% endblock %}
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\templates\index.html", line 11, in block 'app_content'
    {% include '__post.html' %}
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\templates\__post.html", line 36, in top-level template code
    <a href="{{ url_for('main.user', username=post.author.username) }}">
  File "c:\users\anton\downloads\lib\site-packages\flask\helpers.py", line 338, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "c:\users\anton\downloads\lib\site-packages\flask\helpers.py", line 325, in url_for
    rv = url_adapter.build(
  File "c:\users\anton\downloads\lib\site-packages\werkzeug\routing.py", line 2305, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'main.user' with values ['username']. Did you mean 'user' instead?
2021-09-26 11:26:52,893 INFO: Howler startup [in C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\__init__.py:60]
2021-09-26 11:27:24,649 INFO: Howler startup [in C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\__init__.py:60]
2021-09-26 11:27:26,971 ERROR: Exception on / [GET] [in c:\users\anton\downloads\lib\site-packages\flask\app.py:1454]
Traceback (most recent call last):
  File "c:\users\anton\downloads\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\anton\downloads\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\anton\downloads\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\anton\downloads\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "c:\users\anton\downloads\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\routes.py", line 53, in index
    return render_template('index.html', title=_('Home'), form=form,
  File "c:\users\anton\downloads\lib\site-packages\flask\templating.py", line 147, in render_template
    return _render(
  File "c:\users\anton\downloads\lib\site-packages\flask\templating.py", line 128, in _render
    rv = template.render(context)
  File "c:\users\anton\downloads\lib\site-packages\jinja2\environment.py", line 1304, in render
    self.environment.handle_exception()
  File "c:\users\anton\downloads\lib\site-packages\jinja2\environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\templates\index.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\anton\downloads\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\anton\downloads\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block 'doc'
    {%- block html %}
  File "c:\users\anton\downloads\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block 'html'
    {% block body -%}
  File "c:\users\anton\downloads\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 23, in block 'body'
    {% block content -%}
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\templates\base.html", line 48, in block 'content'
    {% block app_content %}{% endblock %}
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\templates\index.html", line 11, in block 'app_content'
    {% include '__post.html' %}
  File "C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\templates\__post.html", line 36, in top-level template code
    <a href="{{ url_for('user.username', username=post.author.username) }}">
  File "c:\users\anton\downloads\lib\site-packages\flask\helpers.py", line 338, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "c:\users\anton\downloads\lib\site-packages\flask\helpers.py", line 325, in url_for
    rv = url_adapter.build(
  File "c:\users\anton\downloads\lib\site-packages\werkzeug\routing.py", line 2305, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'user.username' with values ['username']. Did you mean 'user' instead?
2021-09-26 11:27:58,579 INFO: Howler startup [in C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\__init__.py:60]
2021-09-26 11:31:00,337 INFO: Howler startup [in C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\__init__.py:60]
2021-09-26 11:33:42,416 INFO: Howler startup [in C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\__init__.py:60]
2021-09-26 11:42:07,388 INFO: Howler startup [in C:\Users\anton\Downloads\Dissertation Folder\NewHowler-5afd7201a232c3e73e2854027d7564bcaaef174f\app\__init__.py:60]
